README.md

```diff
@@ -0,0 +1,19 @@
+# Fred
+
+MDN's next fr(ont)e(n)d.
+
+## Getting started
+
+> [!NOTE]
+> You only need Node.js installed. If you don't have globally installed yarn, you can run commands as `npx yarn` instead of `yarn`.
+
+1. In the [yari](https://github.com/mdn/yari/) repository:
+  - Install dependencies `yarn`
+  - Start backend `yarn rari serve`
```

Comment:
We have rari available in fred now with `npm run rari`, so this bit can be rewritten

Reply:
Set `CONTENT_ROOT` to the path of your content, e.g:

`CONTENT_ROOT=../content/files npm run rari serve`